src/components/icon-card-link-grid-list/types.ts

```diff
@@ -6,7 +7,8 @@ export type IconCard = Pick<
 	'icon' | 'productSlug' | 'text' | 'url'
 >
 
-export interface IconCardLinkGridListProps {
+export interface IconCardLinkGridListProps
+	extends Pick<CardsGridListProps, 'gridGap' | 'fixedColumns'> {
```

Comment:
`IconCardLinkGridList` now supports `gridGap and `fixedColumns`, which are passed through to `CardsGridList`.